APMrover2/crash_check.cpp

```diff
@@ -1,9 +1,9 @@
 #include "Rover.h"
 
 // Code to detect a crash or block
-#define CRASH_CHECK_TRIGGER_SEC     2       // 2 seconds blocked indicates a crash
-#define CRASH_CHECK_THROTTLE_MIN    5.0f    // vehicle must have a throttle greater that 5% to be considered crashed
-#define CRASH_CHECK_VEL_MIN         0.08f   // vehicle must have a velocity under 0.08 m/s or rad/s to be considered crashed
+static const uint16_t CRASH_CHECK_TRIGGER_SEC = 2;   // 2 seconds blocked indicates a crash
+static const float CRASH_CHECK_THROTTLE_MIN = 5.0f;  // vehicle must have a throttle greater that 5% to be considered crashed
+static const float CRASH_CHECK_VEL_MIN = 0.08f;      // vehicle must have a velocity under 0.08 m/s or rad/s to be considered crashed
 
```

Comment:
No change in size : 
static const version
````
BUILD SUMMARY
Build directory: /home/khancyr/Workspace/ardupilot/build/sitl
Target         Text     Data  BSS    Total  
--------------------------------------------
bin/ardurover  1106343  1888  29408  1137639
````
Define version 
````
BUILD SUMMARY
Build directory: /home/khancyr/Workspace/ardupilot/build/sitl
Target         Text     Data  BSS    Total  
--------------------------------------------
bin/ardurover  1106343  1888  29408  1137639
````
unless I don't look at the good thing